packages/block-editor/src/components/inserter/style.scss

```diff
@@ -268,15 +268,11 @@ $block-inserter-tabs-height: 44px;
 	}
 }
 
-.block-editor-inserter__block-patterns-tabs-container,
-.block-editor-block-patterns-explorer__sidebar {
+.block-editor-inserter__block-patterns-tabs-container {
```

Comment:
This change will also remove the following selector.

```css
.block-editor-block-patterns-explorer__sidebar nav {
  height:100%;
}
```

However, it seems that there are no elements matching this selector.